Formula/mos.rb

```diff
@@ -4,10 +4,14 @@ class Mos < Formula
   # update_hb begin
   desc "Mongoose OS command-line tool"
   homepage "https://mongoose-os.com/"
-  url "https://github.com/mongoose-os/mos/archive/1ec85951e77649f1151bc06a12901faa784a991a.tar.gz"
-  sha256 "1d95538be29219a487ec21724f65ca9e4ab28341e2a722ac6dd6abd249560583"
-  version "2.18.0"
-  head ""
+  url "https://github.com/mongoose-os/mos/archive/2.18.0.tar.gz"
+  sha256 "46fa3902878dee9e21a94b2686b084f90f517a9175aa20b81d984811b227ef56"
+  license "Apache-2.0"
```

Comment:
should add this to mos-latest as well

Reply:
Good catch @rojer. I've added the license to mos-latest